frontend/src/Authentication.css

```diff
@@ -0,0 +1,11 @@
+@media all and (min-width: 480px) {
```

Comment:
it will be better if we use some predefined grids than take care about media query. Let's use bootstrap and col-sm -md -lg styles